stackslib/src/chainstate/stacks/boot/mod.rs

```diff
@@ -1165,6 +1166,37 @@ impl StacksChainState {
             });
         Ok(aggregate_public_key)
     }
+
+    // Get the current signer set
+    pub fn get_current_signer_set_pox_4(
```

Comment:
I don't think you currently have the means of testing this, because there isn't any logic written to write state into `reward-cycle-signing-keys`.  So, I think one of two things need to happen:

* You can add the logic to insert state into `reward-cycle-signing-keys`, e.g. via a private function in `pox-4` that the node can call.  If you take this route, then you should also add some code to actually populate this data map so you can proceed to test this function in a unit test.

* You can delete this function for now, and we'll add it back in once it's possible to populate `reward-cycle-signing-keys`.